src/gql/fragments/projectSettings/githubCommitQueue.graphql

```diff
@@ -35,3 +35,41 @@ fragment repoGithubCommitQueue on RepoSettings {
     }
   }
 }
+
+fragment projectEventGithubCommitQueue on ProjectEventSettings {
+  githubWebhooksEnabled
+
+  projectRef {
+    prTestingEnabled
+    githubChecksEnabled
+    githubTriggerAliases
+    gitTagVersionsEnabled
+    gitTagAuthorizedUsers
+    gitTagAuthorizedTeams
+    commitQueue {
+      enabled
+      requireSigned
+      mergeMethod
+      message
+    }
+  }
```

Comment:
[nit] It may be preferable to make two fragments on `Project` and `RepoRef` to use within this file... then we can repeat the same thing only twice instead of 4x 😅

Reply:
good point, thanks!